### ISSUE TITLE
Validation Errors CSS, WCAG 2.0, Section 508 (2017)

### ISSUE THREAD

**Issue by rich-eicher** [webgrafia/ginger#29]
I use powermapper.com to audit our sites for Standards and Accessibility compatibility and compliance. Here are a few issues it detected with Gingers CSS

Priority 1 Ensure that all information conveyed with color is also available without color, for example from context or markup. This style only contains a color change. Ensure that each CSS style contains style properties other than color and background-color.

Line 126 WCAG 2.0 A 1.4.1 Section 508 (2017) A 1.4.1

126            .ginger_container.light{
127             background-color: #f7f9fc;             color: #03005b;            }

Priority 2 If you set any of the colors on the BODY or A elements you must set all of them.
In HTML the color attributes are TEXT, BGCOLOR, LINK, ALINK and VLINK. In CSS the attributes are COLOR and BACKGROUND-COLOR. Some users have browser defaults set to white text on a black background, so setting one color without setting the others can result in black text on a black background. High contrast mode on Windows turns off background images so you must set a background-color in addition to a background-image.

Line 128 WCAG 2.0 AA F24 Section 508 (2017) AA F24

128                        a.ginger_btn.ginger-accept, a.ginger_btn.ginger-disable, .ginger_btn{

129                background: #d1d1d1 !important;

130            }

Priority 2If you set any of the colors on the BODY or A elements you must set all of them.
In HTML the color attributes are TEXT, BGCOLOR, LINK, ALINK and VLINK. In CSS the attributes are COLOR and BACKGROUND-COLOR. Some users have browser defaults set to white text on a black background, so setting one color without setting the others can result in black text on a black background. High contrast mode on Windows turns off background images so you must set a background-color in addition to a background-image.
Line 131 WCAG 2.0 AA F24 Section 508 (2017) AA F24
131            a.ginger_btn.ginger-accept:hover, a.ginger_btn.ginger-disable:hover, .ginger_btn{

132                background: #d1d1d1 !important;

133            }

Priority 1Ensure that all information conveyed with color is also available without color, for example from context or markup. This style only contains a color change.
Ensure that each CSS style contains style properties other than color and background-color.
Line 134 WCAG 2.0 A 1.4.1 Section 508 (2017) A 1.4.1
Priority 2If you set any of the colors on the BODY or A elements you must set all of them.
In HTML the color attributes are TEXT, BGCOLOR, LINK, ALINK and VLINK. In CSS the attributes are COLOR and BACKGROUND-COLOR. Some users have browser defaults set to white text on a black background, so setting one color without setting the others can result in black text on a black background. High contrast mode on Windows turns off background images so you must set a background-color in addition to a background-image.

Line 134 WCAG 2.0 AA F24 Section 508 (2017) AA F24

134                                    a.ginger_btn {

135                color: #0f0f0f !important;

136            }

137                                    .ginger_container.light a{

138             color: #0060b5;            }

Priority 1 CSS Validation Error.
Line 139 CSS Validation
139                                    {

140     padding:20px 20px;

141}                    </style>
